content/en/docs/reference/glossary/pod-disruption-budget.md

```diff
@@ -22,5 +22,5 @@ tags:
 
 <!--more--> 
 
-PDBs cannot prevent an involuntary disruption, but 
- will count against the budget.
+PDBs cannot prevent an involuntary disruption. However, involuntary disruptions 
```

Comment:
Possible rephrase to below because involuntary disruptions seems to be repeated too many times?

Involuntary disruptions cannot be prevented by PDBs; however they do count against the budget.

Reply:
\(^-^)/ I made the change that you requested, Divya! 

Once this is finalized I will submit a PR for a rewrite of the same line under /docs/concepts/workloads/pods/disruptions